tests/src/Controller/AccessTokenControllerTest.php

```diff
@@ -28,7 +30,7 @@ class AccessTokenControllerTest extends TestCase
     protected function setUp(): void
     {
         $this->authorizationServerMock = $this->createMock(AuthorizationServer::class);
-
+        $this->allowedOriginRepository = $this->createMock(AllowedOriginRepository::class);
```

Comment:
AccessTokenControllerTest  needs a test to confirm it is invoking the CORs functionality. I think copying `testItHandlesCorsRequest` from `UserInfoControllerTest` will handle that.

@cicnavi I'm curious on your thoughts about the test methods `testItThrowsIfCorsOriginNotAllowed` and `testItThrowsIfOriginHeaderNotAvailable`:  should we leave those in `UserInfoControllerTest`  or create a test case class for the new `RequestTrait` and move those tests there.

Reply:
Added the additional test. Waiting for @cicnavi to provide feedback to the `RequestTrait` test class.

Reply:
Since those test methods ensure behavior of method 'handleCors' in the trait, it makes sense to do it separately... PHPUnit has method 'getObjectForTrait' which could be used for easy creation of instance which uses the trait, but it has been deprecated. We can also use anonymous classes which uses the trait to only test the trait... Me personally, I'm not a big fan of traits, I would rather create helper / service class instead. But we already have traits in codebase, so it's ok.

Reply:
Moved the tests in a new class that is a wrapper around the trait.